README.md

```diff
@@ -85,16 +85,38 @@ Note that non-default networks require extra [firewall setup](https://cloud.goog
   [nfs-client](https://github.com/kubernetes-incubator/external-storage/tree/master/nfs-client)
   external provisioner can be used to provide similar functionality for
   Kubernetes clusters.
+* Windows support. The current version of the driver supports volumes mounted to Linux nodes only.
+
+## Deploying the Driver
+
+* Set up a service account with appropriate role binds, and download a service account key. This
+  service account will be used by the driver to provision Filestore instances and otherwise access
+  GCP APIs. This can be done by running `./deploy/project_setup.sh` and pointing to a directory to
+  store the SA key. To prevent your key from leaking do not make this directory publicly
+  accessible!
+  
+```$ PROJECT=<your-gcp-project> GCFS_SA_DIR=<your-directory-to-store-credentials-by-default-home-dir> ./deploy/project_setup.sh```
+
+* Choose a stable overlay that matches your cluster version, eg `stable-1-19`. If you are running a
+  more recent cluster version than given here, use `stable-master`. The `prow-*` overlays are for
+  testing, and the `dev` overlay is for driver development. `./deploy/kubernetes/cluster-setup.sh`
+  will install the driver pods, as well as necessary RBAC and resources.
+
+```$ PROJECT=<your-gcp-project> DEPLOY_VERSION=<your-overlay-choice> GCFS_SA_DIR=<your-directory-to-store-credentials-by-default-home-dir> ./deploy/kubernetes/cluster_setup.sh ```
```

Comment:
seems need next line after ```? otherwise format is not right?

Reply:
Thanks, none of that was consistent. Fixed.